lib/AppInfo/Application.php

```diff
@@ -62,70 +57,53 @@ public function __construct(array $urlParams = [])
 
         $container = $this->getContainer();
 
+        \OC::$server->registerService(AllFeeds::class, function (IContainer $c) {
+            return new AllFeeds($c->query(FeedService::class));
+        });
+
         // files
-        $container->registerService('checksums', function () {
+        \OC::$server->registerService('checksums', function () {
```

Comment:
Why migrate from `$container` to `\OC::$server`? 

`$container = $this->getContainer();` is a `IAppContainer` instance
`\OC::$server` is a `ServerContainer` instance